src/Subscriber.ts

```diff
@@ -247,18 +247,18 @@ class SafeSubscriber<T> extends Subscriber<T> {
     }
   }
 
-  private __tryOrUnsub(fn: Function, value?: any): void {
+  private __tryOrUnsub(fn: Function, ...value: Array<any>): void {
     try {
-      fn.call(this._context, value);
+      fn.call(this._context, ...value);
```

Comment:
I can, it's eventually same. I just picked rest operator for simplicity of readability.